.github/workflows/build.yml

```diff
@@ -419,3 +432,44 @@ jobs:
         GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
         PR_URL: ${{github.event.pull_request.html_url}}
       run: gh pr merge --auto --rebase "$PR_URL"
+
+
+  ###########################################################
+  merge-coverage:
```

Comment:
I am not sure if that even is required. maybe just use coverage from one of the jobs and control which job via a matrix parameter?